tsconfig.json

```diff
@@ -1,3 +1,6 @@
 {
-  "extends": "@tsconfig/node20/tsconfig.json"
+  "extends": "@tsconfig/recommended/tsconfig.json",
+  "compilerOptions": {
+    "target": "ES2023"
```

Comment:
To fix the following error:

```console
    src/namespace.ts:16:23 - error TS2550: Property 'replaceAll' does not exist on type 'string'. Do you need to change your target library? Try changing the 'lib' compiler option to 'es2021' or later.

    16     content = content.replaceAll(placeholder, v)
                             ~~~~~~~~~~
```

https://github.com/quipper/monorepo-deploy-actions/actions/runs/10037423243/job/27737172591